src/mscorlib/src/System/Threading/WaitHandle.cs

```diff
@@ -448,7 +448,7 @@ public static int WaitAny(WaitHandle[] waitHandles, int millisecondsTimeout)
                                         WaitHandle toWaitOn)
         {
 #if PLATFORM_UNIX
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(WHAT_TEXT_HERE);
```

Comment:
These are all the SignalAndWait and in a previous discussion with @jkotas and @kouvel we determined we could implement these for Unix, so I'll open an issue.